projects/core/src/lib/operators/dialogs/rgb-composite/rgb-composite.component.html

```diff
@@ -0,0 +1,87 @@
+<geoengine-sidenav-header>Calculate Expression on Raster</geoengine-sidenav-header>
```

Comment:
is still "Expression"

Reply:
[e41152d](https://github.com/geo-engine/geoengine-ui/pull/647/commits/e41152d0d1e578567e370d37b8d1d10456999d72)